### PR TITLE
Track page ID of clicked paid card in related container

### PR DESCRIPTION
**Pull request by kelvin-chappell**
Following on from #18044, this change will give enough information to track referrals from related containers in other sections.

Eg.
Before:
> Onward container | UK | unknown front id | container-2 | related content | NAB | card-8 | A bug’s life for sustainability? It’s only natural, says top wine executive

After:
> Onward container | UK | nab-more-that-matters/2017/jun/27/shared-value-how-business-is-stepping-up | container-2 | related content | NAB | card-8 | A bug’s life for sustainability? It’s only natural, says top wine executive

### CODE REVIEW

File: common/app/implicits/Requests.scala [hunk 1]
```diff
@@ -45,6 +45,8 @@ trait Requests {
     lazy val campaignCode: Option[String] = r.getQueryString("CMP")
 
     lazy val isAdFree: Boolean = r.headers.keys.exists(_ equalsIgnoreCase "X-Gu-Commercial-Ad-Free")
+
+    lazy val referrer: Option[String] = r.headers.get("referer")
```
Comment (LATaylor-guardian) [inline common/app/implicits/Requests.scala]: is the header name `referer` or `referrer`?

**Comment by LATaylor-guardian** [comment]
👍

File: common/app/implicits/Requests.scala (hunk 1 above)
Comment (kelvin-chappell) [reply common/app/implicits/Requests.scala]: Madly, it's called `referer`.
https://en.wikipedia.org/wiki/HTTP_referer#cite_note-1

Comment (janua) [reply common/app/implicits/Requests.scala]: 😂

**Review by rich-nguyen — APPROVED** [review]

**Pull request merged**